generated/nidaqmx/system/watchdog.py

```diff
@@ -77,7 +78,8 @@ def __enter__(self):
         return self
 
     def __exit__(self, type, value, traceback):
-        self.close()
+        if self._is_new_session_initialized:
```

Comment:
This is interesting... The alternative to close is `detach`, a new method, which leaks the session on the server (intentionally), so we wouldn't call close. That being said, I just checked nimi-python, and this is what it does, too. So nevermind :D

Reply:
Actually, nimi-python calls the member variable `_close_on_exit` which is a bit more clear. In the grpc interpreter they do this:

```python
self._close_on_exit = response.new_session_initialized
```

Then the top-level tasks/sessions check the interpreter's value of `close_on_exit` to close or not.

Reply:
Thanks for the information! Updated the name of `_close_on_exit`.